src/components/global/TaskModalStyles.tsx

```diff
@@ -0,0 +1,119 @@
+import CloseIcon from '@mui/icons-material/Close'
```

Comment:
Why don't we move this file (and other styles-only files) to a new folder `src/styles` :)